src/test/resources/testdata/fixtures/api.omise.co/disputes/dspt_test_5089off452g5m5te7xs-get.json

```diff
@@ -12,5 +12,9 @@
   "metadata": {
     "description": "DESCRIPTION",
     "invoice_id": "inv_N1ayTWJ2FV"
-  }
+  },
+  "transaction": "trxn_test_59bqoovjrgehphloams",
+  "reason_code": "goods_or_services_not_provided",
+  "reason_message": "Services not provided or Merchandise not received",
+  "closed_at": null
```

Comment:
Should we add some date to test the date can serializable?

Reply:
@nuxzero Sure, Added 🙇